modules/apps/dynamic-data-mapping/dynamic-data-mapping-uad-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/uad/util/DDMFormInstanceRecordUADTestUtil.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.dynamic.data.mapping.uad.util;
+
+import com.liferay.dynamic.data.mapping.model.DDMForm;
+import com.liferay.dynamic.data.mapping.model.DDMFormInstance;
+import com.liferay.dynamic.data.mapping.model.DDMFormInstanceRecord;
+import com.liferay.dynamic.data.mapping.model.DDMFormInstanceRecordVersion;
+import com.liferay.dynamic.data.mapping.service.DDMFormInstanceRecordLocalService;
+import com.liferay.dynamic.data.mapping.storage.DDMFormValues;
+import com.liferay.dynamic.data.mapping.test.util.DDMFormTestUtil;
+import com.liferay.dynamic.data.mapping.test.util.DDMFormValuesTestUtil;
+import com.liferay.portal.kernel.model.Group;
+import com.liferay.portal.kernel.service.ServiceContext;
+import com.liferay.portal.kernel.test.util.ServiceContextTestUtil;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.workflow.WorkflowConstants;
+
+/**
+ * @author Gabriel Ibson
+ */
+public class DDMFormInstanceRecordUADTestUtil {
```

Comment:
@marcosapmf, the util classes `DDMFormInstanceRecordUADTestUtil` and `DDMFormInstanceUADTestUtil` are currently being used by the GDPR code and, probably, for this reason their names are related to GDPR, however these classes are only related to `DDMFormInstanceRecord` and `DDMFormInstance`, respectively. Therefore, we should rename these classes to make clear that they're used by GDPR code, but are not related to it and we should avoid to send a `LocalService` instance as a parameter, honestly, I feel that Brian won't like this, please check the class `DDMStructureTestUtil` to get some inspiration to find out how to use a `LocalService` instance in an util class. If you prefer, you can also remove these util classes and add all code you need in the tests which are consuming these util classes. In light of this comment, can you please re-send?